src/widgets/pictures/ui/pictures.scss

```
@@ -1,85 +1,46 @@
-.pictures {
-  & .social-media-container {
-    @include media-tablet {
-      flex-wrap: wrap;
-      align-items: center;
-      justify-content: center;
-    }
-
-    display: flex;
-    align-items: flex-start;
-  }
-}
-
-.pictures .carousel-root {
-  @include media-laptop {
-    width: 100%;
-    max-width: 730px;
-    height: 369px;
-  }
-
-  @include media-tablet {
-    width: 100%;
-    max-width: 358px;
-    height: 210px;
-    padding: 0 0 25px;
-  }
-
-  @include media-mobile {
-    width: 100%;
-    height: auto;
+.carousel {
+  .slide {
+    cursor: pointer;
+    padding: 0 5px;
   }
 
-  width: 1200px;
-  height: 540px;
-  margin: 16px auto;
-  padding: 10px 10px 40px;
-}
+  .carousel-slider {
+    .control-next,
+    .control-prev {
+      top: calc(50% - 50px);
```

please move the magic number to a variable with meaningful name

```suggestion
      --some-value: 50px;
      top: calc(50% - var(--some-value));
```

---

Done